Kursovaya_frst_part

qwellqy opened the pull request:
1_chast'

qwellqy committed:
Add files via upload

1_chast'

ShumilinDmA commented:
Нужно

1) Декомпозировать код на функции
2) Использовать ваш собстенный класс водяного пара из первой домашней работы

qwellqy committed:
Add files via upload

Постарался декомпозировать класс на функции, как только Вы апрувните, я сразу его подключу в курсовую вместо того, что я Вам прислал в ветке Kursovaya_part_2;
Я уже пробовал его подключать, ничего не слетает, все ок и считает аналогично;
осталось только, чтобы Вы проверили

qwellqy commented:
Добавил новый st файл (декомпозировал на функции для улучшения читаемости);
Как только Вы его апрувните, я сразу его подключу в курсовую;
я уже пробовал его подключать вместо того, что я Вам прислал в ветке Kursovaya_part_2, все считает, все окей, никак не ломается;
осталось только Вам его проверить;

Также подключил свой собственный класс для расчета вместо использования iapws

ShumilinDmA commented:
Норм, но нет кода для пара. Что ты импортируешь и используешь его я вижу. Но в гите его тут нет
